README.md

```diff
@@ -117,18 +116,7 @@ pip install ghostscript camelot-py
 4. For table detection, install [**camelot**](https://camelot-py.readthedocs.io/en/master/user/install-deps.html#for-windows).
 5. For the **pdfminer** extractor for pdfs, [follow these steps](https://github.com/pdfminer/pdfminer.six#how-to-install).
```

Comment:
This line should be at number 2 or 3 in the list.

Reply:
I just figured that in README_zh-cn this list is incomplete